R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
Yup, this is correct. Though it should be noted we need to be careful that both the beginning and ending time step is supplied (so midnight day 0 and midnight day 1). For example, would be all too easy to supply midnight day 0 to 11PM day 0 for hourly data, not supplying the last relevant time step (midnight) for the day. (though also important to note this is a very small difference when sampling frequency is higher, the example here has only 5 observations, exacerbating the difference).